lib/node_modules/@stdlib/math/stats/anova1/examples/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var anova1 = require('../lib/make_anova.js');
+
+var x = [3, 4, 5, 6, 2, 5, 10, 12, 8, 10];
+var f = ['frank', 'philipp', 'nugent', 'frank', 'philipp', 'nugent', 'frank', 'philipp', 'nugent', 'frank'];
```

Comment:
Let's change the example to also use labels such as `Treatment` and `Control`.

Reply:
Resolved.